scripts/run-ipv6-integration-tests.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# The script runs the full IPv6 integration test suite
+
+set -e
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+GINKGO_TEST_BUILD="$SCRIPT_DIR/../test/build"
+
+source "$SCRIPT_DIR"/lib/add-on.sh
+source "$SCRIPT_DIR"/lib/cluster.sh
+source "$SCRIPT_DIR"/lib/canary.sh
+
+function run_ginkgo_test() {
+  (CGO_ENABLED=0 ginkgo $EXTRA_GINKGO_FLAGS -v --timeout 30m --no-color --fail-on-pending $GINKGO_TEST_BUILD/ipv6.test -- --cluster-kubeconfig="$KUBE_CONFIG_PATH" --cluster-name="$CLUSTER_NAME" --aws-region="$REGION" --aws-vpc-id="$VPC_ID" --ng-name-label-key="kubernetes.io/os" --ng-name-label-val="linux")
```

Comment:
Does this test complete in 30 mins?
If so, should we run full ipv6 test suite alone and remove the ipv6 canary script? Given that the tests are not flaky.

Reply:
Yeah, it completes in < 10 minutes, and the plan is to remove ipv6 canary script, but in order to do in CI manner, I think we have to have both, point beta to new, point prod to new, remove old